metrics/src/metrics.rs

```diff
@@ -53,7 +55,7 @@ macro_rules! datapoint {
     };
 
     ($name:expr, $($fields:tt)+) => {
-        $crate::submit($crate::datapoint!(@point $name, $($fields)+));
+        $crate::submit($crate::datapoint!(@point $name, $($fields)+), log::Level::Info);
```

Comment:
How about allowing folks to pass level in?  Could also have a pattern where the default level is `Info`

Reply:
datapoint_error/datapoint_warn/datapoint_info/datapoint_debug/datapoint_verbose?

eg:
```bash
datapoint_info!("vote-native", ("count", 1, i64));
```

Reply:
Sounds good. I am adding similar macros for inc_new_counter_**. I'll add for datapoint! as well.
That's a separate PR though, as it's touching a lot of files.